neo/SmartContract/Native/Votes/Method/FindAllPathMethod.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract.Native
+{
+    public class FindAllPathMethod
+    {
+        /// <summary>
+        /// Node Set
+        /// </summary>
+        private List<String> nodesList = new List<String>();
+        /// <summary>
+        /// Vector Graphic,describe the connection between nodes and nodes
+        /// </summary>
+        private Dictionary<String, int> vectorGraphic = new Dictionary<String, int>();
+        /// <summary>
+        /// Edge Array,describe the connection between nodes and nodes
+        /// </summary>
+        private int[,] edgeArray;
+        /// <summary>
+        /// Path Set,store paths in vector graphic.
+        /// </summary>
+        private List<List<String>> path = new List<List<String>>();
```

Comment:
Let's give `List<List<String>>` a typedef to something, like `nodesPath`, the vector as assymetric, right? Each item can have a distinct size of the path, correct?

Is not the path an integer? Why do we need String? Better to link everything to an index, no?

Reply:
Just a habit.You know ,in java,remove function has 2 implement：“remove(int index)” and "remove(Object obj)".It is very inconvenient to understand when it define a List<int> type.
But in c#,this problem does not exist.

Reply:
If you think it is not inappropriate,I will change it